### DISCUSSION TITLE
amamba/user-guide/wizard/jar-java-app/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1767] [Announcements]
# amamba/user-guide/wizard/jar-java-app/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/user-guide/wizard/jar-java-app/

**Comment by liyuerich** [comment]
应用组 是什么概念，能具体介绍一下吗？

**Comment by peterchenhc** [comment]
daocloud-demo.jar 下载链接已经失效
